### PR TITLE
Added deprecation warning for prisma introspect command

### LINKED ISSUE #2115: Update docs to reflect that `prisma introspect` gets deprecated in favor of `prisma db pull`

### PR DESCRIPTION
**Pull request by molebox**

### CODE REVIEW

**Comment by thebiglabasky** [comment]
I'd move the deprecation warning from the reference section up, just below the header `introspect`

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 1]
```diff
@@ -420,6 +420,13 @@ datasource db {
 
 ##### Analyze the database and write its schema to the `schema.prisma` file
 
+<Admonition type="warning">
+
+**Deprecation warning** <br />
+From Prisma 3.0.0 onwards, the `prisma introspect` command will be deprecated and replaced with the [`prisma db pull`](#db-pull) command.
+
+</Admonition>
+
```
Comment (Jolg42) [inline content/400-reference/200-api-reference/200-command-reference.mdx]: Could make sense to move the warning and have it next to "**MongoDB not supported**" I think

File: content/400-reference/200-api-reference/100-prisma-schema-reference.mdx [hunk 2]
```diff
@@ -875,6 +875,13 @@ The [MongoDB connector](../../concepts/database-connectors/mongodb) does not cur
 
 The `Unsupported` type was introduced in [2.17.0](https://github.com/prisma/prisma/releases/tag/2.17.0) and allows you to represent data types in the Prisma schema that are not supported by the Prisma Client. `Unsupported` fields can be introspected with `prisma introspect` or created with Prisma Migrate or `db push`.
 
+<Admonition type="warning">
+
+**Deprecation warning** <br />
+From Prisma 3.0.0 onwards, the `prisma introspect` command will be deprecated and replaced with the [`prisma db pull`](./command-reference#db-pull) command.
+
+</Admonition>
+
```
Comment (Jolg42) [inline content/400-reference/200-api-reference/100-prisma-schema-reference.mdx]:
```suggestion
The `Unsupported` type was introduced in [2.17.0](https://github.com/prisma/prisma/releases/tag/2.17.0) and allows you to represent data types in the Prisma schema that are not supported by the Prisma Client. `Unsupported` fields can be introspected with `prisma db pull` or created with Prisma Migrate or `db push`.

```

I think we don't need a warning here, we could just rename?

File: content/300-guides/600-general-guides/100-database-workflows/05-foreign-keys/03-sqlite.mdx [hunk 3]
```diff
@@ -120,6 +120,13 @@ Congratulations, you just created two tables called `AnotherUser` and `AnotherPo
 
 ## 4. Introspect your database with Prisma
 
+<Admonition type="warning">
+
+**Deprecation warning** <br />
+From Prisma 3.0.0 onwards, the `prisma introspect` command will be deprecated and replaced with the [`prisma db pull`](../../../../reference/api-reference/command-reference#db-pull) command.
+
+</Admonition>
+
```
Comment (Jolg42) [inline content/300-guides/600-general-guides/100-database-workflows/05-foreign-keys/03-sqlite.mdx]: I think we don't need a warning here, we could just rename?

**Comment by Jolg42** [comment]
Started to add comments like
`I think we don't need a warning here, we could just rename?`
But I think it applies in many places so stopping commenting to not spam

File: src/components/customMdx/admonition.tsx [hunk 4]
```diff
@@ -16,20 +16,39 @@ const colorMap: any = {
 }
 
 const Admonition = ({ children, type, ...props }: AdmonitionProps) => {
+  console.log({children})
+  const isMultiline: boolean = Array.isArray(children) ? true : false;
```
Comment (Jolg42) [inline src/components/customMdx/admonition.tsx]:
```suggestion
  const isMultiline: boolean = Array.isArray(children) ? true : false;
```
I think we should remove the console.log?

File: content/200-concepts/100-components/04-introspection.mdx [hunk 5]
```diff
@@ -42,9 +42,16 @@ You can learn more about how Prisma maps types from the database to the types av
...
 
-You can introspect your database using the `prisma introspect` command of the [Prisma CLI](prisma-cli/installation). Note that using this command requires your [connection URL](../../reference/database-reference/connection-urls) to be set in your Prisma schema!
+<Admonition type="warning">
+
+**Deprecation warning** <br />
+From Prisma 3.0.0 onwards, the `prisma introspect` command will be deprecated and replaced with the [`prisma db pull`](../../../reference/api-reference/command-reference#db-pull) command. The following will use the `prisma db pull` command.
+
+</Admonition>
+
+You can introspect your database using the `prisma db pull` command of the [Prisma CLI](prisma-cli/installation). Note that using this command requires your [connection URL](../../reference/database-reference/connection-urls) to be set in your Prisma schema!
```
Comment (Jolg42) [inline content/200-concepts/100-components/04-introspection.mdx]:
```suggestion
<Admonition type="warning">

**Deprecation warning** <br />
From Prisma 3.0.0 onwards, the `prisma introspect` command will be deprecated and replaced with the [`prisma db pull`](../../../reference/api-reference/command-reference#db-pull) command.

</Admonition>

You can introspect your database using the `prisma db pull` command of the [Prisma CLI](prisma-cli/installation). Note that using this command requires your [connection URL](../../reference/database-reference/connection-urls) to be set in your Prisma schema!
```

If we rename the title to 
```
## The `prisma db pull` command
```

then we can remove that sentence I think?

**Comment by Jolg42** [comment]
Note: some images are showing `prisma introspect`
Like 
- https://imgur.com/f7itiYw.png from https://www.prisma.io/docs/concepts/components/introspection
- https://imgur.com/ToNkpb2.png from https://www.prisma.io/docs/concepts/components/introspection and https://www.prisma.io/docs/concepts/overview/what-is-prisma

Not sure how we can update these images (maybe for another day)

**Review by Jolg42 — APPROVED** [review]
Looks good to me! 🙌🏼 (see comments before merging)

**Comment by molebox** [comment]
> Note: some images are showing `prisma introspect`
> Like
> 
> * https://imgur.com/f7itiYw.png from https://www.prisma.io/docs/concepts/components/introspection
> * https://imgur.com/ToNkpb2.png from https://www.prisma.io/docs/concepts/components/introspection and https://www.prisma.io/docs/concepts/overview/what-is-prisma
> 
> Not sure how we can update these images (maybe for another day)

Yep i saw those! But thanks for making a list of them, thats super helpful. Will create another issue. I can probably handle most of them so long as they live in figma

File: src/components/customMdx/admonition.tsx (hunk 4 above)
Comment (molebox) [reply src/components/customMdx/admonition.tsx]: face-palm

**Pull request merged**